cli/src/main/java/com/liferay/blade/cli/command/CreateCommand.java

```diff
@@ -72,6 +73,16 @@ public CreateCommand(BladeCLI bladeCLI) {
 	public void execute() throws Exception {
 		CreateArgs createArgs = getArgs();
 
+		File argsDir = createArgs.getDir();
+
```

Comment:
this you should check the baseDir of create args. The argsDir should be project dir.  Please add test.